src/truefoundry_sdk/core/jsonable_encoder.py

```
@@ -106,3 +106,15 @@ def fallback_serializer(o: Any) -> Any:
         return jsonable_encoder(data, custom_encoder=custom_encoder)
 
     return to_jsonable_with_fallback(obj, fallback_serializer)
+
+
+def encode_path_param(obj: Any) -> str:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `obj`

(ANN401)